source/_components/sensor.rmvtransport.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: page
+title: "RMV"
+description: "Instructions on how to integrate Rhein-Main public transport departure times into Home Assistant."
+date: 2018-08-02 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: RMV.png
+ha_category: Transport
+ha_release: 0.76
+ha_iot_class: "Cloud Polling"
+---
+
+The `rvmtransport` sensor will give you the departure time of the next bus, tram, subway, or train at the next station or stop in the Rhein-Main area public transport network. Additional details such as the line number and destination are present in the attributes.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rmvtransport
+    nextdeparture:
```

Comment:
Isn't this called  `next_departure` in the merged version? (Which by the way is not consistent with the MVG sensor, where we have `nextdeparture` and `timeoffset`. Not sure if this should be handled consistently.)

Reply:
Thank's for pointing it out. I agree with your remark about consistency, but it actually is PEP 8 compliant. So it would rather make sense to change the names of the MVG sensor, no?